pkg/sqlite/load.go

```diff
@@ -725,3 +739,74 @@ func (s *sqlLoader) addDependencies(tx *sql.Tx, bundle *registry.Bundle) error {
 
 	return nil
 }
+
+func (s *sqlLoader) addPackageProperty(tx *sql.Tx, bundleName, pkg, version string) error {
```

Comment:
I can see `addPackageProperty` and `addGVKProperties` are for the 2 specific property types. In order to make this generic, should we add a generic func `addProperty` which will take a `type` and a `value`? Then, simply add those values to the `properties` table? Or this part will be addressed in subsequent PR?